src/test/ui/parser/issue-68890.rs

```diff
@@ -0,0 +1,4 @@
+enum e{A((?'a a+?+l))}
```

Comment:
A slightly simpler reproducer btw, `type X = (?'a ?+l);`.

Reply:
OK, I'll update this.

Reply:
Not necessary, yours is sufficiently minimal. ;)